### ISSUE TITLE
Problème DOIs et phrases

### ISSUE THREAD

**Issue by sguillaume** [CNRS/Pangloss#160]
Les DOIs des phrases ne pointent pas sur les phrases concernées mais tous pointent sur le texte au niveau du titre (ça ouvre la page au niveau du titre au lieu d'ouvrir la page sur la phrase citée)

ex : https://doi.org/10.24397/pangloss-0001471#S10

**Comment by alexis-michaud** [comment]
Oui, c'est quelque chose qu'on ne pouvait pas tester avec la maquette. 

Il faut que l'URL générée par le résolveur DOI contienne l'information de l'ancre sur laquelle 'caler' le texte.

Dans cet exemple, c'est le #S10 en fin d'URL : 
[https://pangloss.cnrs.fr/corpus/show?lang=en&mode=pro&oai_primary=cocoon-0260c314-f906-3cf2-8d74-b62d0905b321&oai_secondary=cocoon-b3bbd8b7-2c3b-339c-a380-f0e941d1662f#S10](https://pangloss.cnrs.fr/corpus/show?lang=en&mode=pro&oai_primary=cocoon-0260c314-f906-3cf2-8d74-b62d0905b321&oai_secondary=cocoon-b3bbd8b7-2c3b-339c-a380-f0e941d1662f#S10)

Vous voyez l'idée ? 

Pour tous détails, au cas où vous auriez besoin d'éléments de contexte : infos très détaillées [ici](https://halshs.archives-ouvertes.fr/halshs-02870206/)

**Comment by alexis-michaud** [comment]
Tant qu'on y est, on signale aussi le souci à @m8nli9ht : il y a une info à passer au lecteur, pour qu'il se cale sur l'unité n du premier niveau de granularité : S pour les textes (TEXT), W pour les listes de mots (WORDLIST).

**Comment by m8nli9ht** [comment]
Le soucis c'est que l'on a un autre ID que S10 (généré par le player) pour cette phrase et que l'on récupére depuis le fichier annotations : 
![image](https://user-images.githubusercontent.com/6407776/104425249-bc676680-5580-11eb-9223-be74bff5c448.png)
 
Ce n'est pas la première fois qu'on a une demande pour ignorer un ID, la question est donc : doit-on tenir compte des ID de phrases et de mots renseignés dans les fichiers annotations ? Si non = numérotations incrémentales dans le player, Si oui = dans quel(s) cas et doit-on les afficher quelque part ?

**Comment by alexis-michaud** [comment]
Réponse : **Non, on ne tient pas compte des ID de phrases et de mots renseignés dans les fichiers annotations**. Ils peuvent être erronés : numérotation non continue si une phrase a été insérée manuellement (ou un mot pour les listes de mots). 

On re-numérote en parcourant le XML (_parsing_), en re-numérotant de 1 à n. C'est cette notation qui est affichée (S1, S2...) dans l'interface.

Autrement dit : numérotations incrémentales dans le player, c'est bien ça !
C'est à cette numérotation que fait référence le suffixe ajouté au DOI, et non aux identifiants contenus dans le fichier XML.
